docs/user-guide/cli-using-using-daemon-mode.md

```diff
@@ -73,6 +73,8 @@ You **must** restart daemon mode under the following scenarios:
    You may be running mismatched versions of Zowe executable and Zowe daemon.
    ```
 
+- You created or updated a `zowe.env.json` file in your home directory or the path set in the `ZOWE_CLI_HOME` environment variable. See [Configuring an environment variables file](../user-guide/cli-configuringcli-evfile) for more information.
```

Comment:
I suggest: "...created or updated _the_ `.zowe.env.json` file..." since there would only be one file.

Reply:
updated -- thank you